### ISSUE TITLE
Add "differential mobility"

### ISSUE THREAD

**Issue by sneumann** [HUPO-PSI/psi-ms-CV#179]
### Describe the new term or terms you would like to add.

We need a term `differential mobility`.

**Comment by edeutsch** [comment]
What should be the definition of "differential mobility"? How is it used? How is it different from the existing term "ion mobility separation"?

**Comment by mobiusklein** [comment]
@sneumann Is this equivalent to `FAIMS compensation voltage`/`MS:1001581` for this device, and what should the unit be?

**Comment by sneumann** [comment]
Hi, sorry if the initial issue description was a bit terse, it came out of "Proposed [IM-MS and DIA best practiced document](https://github.com/HUPO-PSI/mzML/blob/master/specification_document/mzML_1.1.0_UseCaseRecommendationForDIAandIMSdata_v0.9.5.docx?raw=true) version 0.9.5" on https://psidev.info/mzML and I extracted a comment and upgraded it to an issue here. 
I myself would not have an answer to your questions right now :-( Yours, Steffen

**Comment by mobiusklein** [comment]
According to https://doi.org/10.1016/j.ijms.2007.07.003, "differential ion mobility" has compensation voltage as its units:
![image](https://user-images.githubusercontent.com/1426041/233796383-266c92c9-7d7f-4540-80cf-5f15e54ff1e6.png)

So we can merge this with #178 and just create a term "SelexION compensation voltage" and be done.

Currently, we don't report an additional `component` in the `instrumentConfiguration` section for FAIMS or even for built-in ion mobility separation. It doesn't have a slot in the `component` model, unless it is a part of the `source` or `analyzer`. 

Do we add just the one term like for FAIMS or try to update the data model to appropriately represent these devices?

**Comment by edeutsch** [comment]
What about just adding the term:

```
[Term]
id: MS:100nnnn
name: SelexION compensation voltage
def: "The voltage applied in the SelexION device to allow certain ions to transmit through to the mass sepctrometer." [PSI:MS]
is_a: MS:1002892 ! ion mobility attribute
is_a: MS:1003254 ! peak attribute
relationship: has_units UO:0000218 ! volt
relationship: has_value_type xsd:float ! The allowed value-type for this CV term
```

https://pdf.medicalexpo.com/pdf/sciex/selexion/79612-207859-_6.html

https://sciex.com/products/ion-mobility-spectrometry/selexion-differential-mobility-separation-device

**Comment by edeutsch** [comment]
addressed in #191
